lms/templates/courseware/courseware.html

```diff
@@ -165,9 +165,6 @@
 
 % if disable_accordion is UNDEFINED or not disable_accordion:
     <div class="course-index" role="navigation">
```

Comment:
@clrux since there is a <nav> element further down in the DOM, we don't need a role defined here.  Especially since it doesn't have a label.  It can be removed all together.